iphone/Classes/TiLogServer.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+
+#ifndef TI_LOG_SERVER_H
+#define TI_LOG_SERVER_H
+
+#ifndef DISABLE_TI_LOG_SERVER
+
+#import <Foundation/Foundation.h>
```

Comment:
`Foundation` is already included in our [prefix header](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/iphone/Titanium_Prefix.pch#L6)